src/main/java/com/microsoft/spring/data/gremlin/conversion/GremlinScriptVertexFindLiteral.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.conversion;
+
+import com.microsoft.spring.data.gremlin.common.Constants;
+import com.microsoft.spring.data.gremlin.exception.UnexpectedGremlinSourceTypeException;
+import lombok.NoArgsConstructor;
+import org.springframework.lang.NonNull;
+import org.springframework.util.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@NoArgsConstructor
+public class GremlinScriptVertexFindLiteral implements GremlinScript<String> {
```

Comment:
`GremlinScriptVertexFindLiteral`, does that mean this class is only used for vertex find query?

Reply:
Yes, the query for find/insert is quit different.